### PR TITLE
DIV-6654: Map general referral states on Petitioner and Respondent apps

### PR DESCRIPTION
**Pull request by madjava**
# Description

https://tools.hmcts.net/jira/browse/DIV-6654

Redirecting PFE to DN when case is in `AwaitingGeneralReferralPayment` or `GeneralConsiderationComplete` state. DN already has facility to display a default informatory page for cases it does not specifically handle, no need to create on PFE

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by yarpo — APPROVED** [review]

File: app/middleware/redirectMiddleware.test.js [hunk 1]
```diff
@@ -65,6 +65,22 @@ describe(modulePath, () => {
       expect(next.calledOnce).to.eql(false);
       expect(res.redirect.calledWith(expectedUrl)).to.eql(true);
     });
+
+    describe('Redirections to DN app', () => {
+      const stateRedirectedToDn = ['AwaitingGeneralReferralPayment', 'GeneralConsiderationComplete'];
```
Comment (jvictorcf) [inline app/middleware/redirectMiddleware.test.js:70]: stateToRedirectToDn?

**Review by jvictorcf — APPROVED** [review]
Approved with a tiny comment.

**Review by jvictorcf — CHANGES_REQUESTED** [review]
Actually. Are these states new (as in not in prod yet)?

File: app/middleware/redirectMiddleware.test.js (hunk 1 above)
Comment (madjava) [reply app/middleware/redirectMiddleware.test.js:70]: Yup, any is fine by me

**Comment by madjava** [comment]
> Actually. Are these states new (as in not in prod yet)?

New states. Features have not yet been released

**Review by jvictorcf — APPROVED** [review]

**Review by yarpo — APPROVED** [review]

**Review by yarpo — APPROVED** [review]

**Pull request merged**